packages/dds/tree/src/test/shared-tree-core/edit-manager/editManagerCorrectness.test.ts

```diff
@@ -639,10 +639,10 @@ export function testCorrectness() {
 		 * - They help diagnose issues with the more complicated exhaustive test (e.g., if one of the above tests fails,
 		 * but this one doesn't, then there might be something wrong with this test).
 		 */
-		describeStress("Combinatorial exhaustive", function ({ isStress }) {
-			const NUM_STEPS = isStress ? 5 : 4;
-			const NUM_PEERS = isStress ? 3 : 2;
-			if (isStress) {
+		describeStress("Combinatorial exhaustive", function ({ stressMode }) {
+			const NUM_STEPS = stressMode !== undefined ? 5 : 4;
```

Comment:
this change makes me think some test suites may only care if stress is enabled or not, and not about the mode that it's in. I'd consider leaving that exposed.